### PR TITLE
Niaz erdoolatov

### PR DESCRIPTION
**Pull request by neojan84**
Swagger Petstore API

### CODE REVIEW

File: api-assignment/petStoreAPITests1.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
+For testing these scenarios we will need: 
+1.Postman or similar tool to perform HTTP requests 
+
+---post---
```
Comment (Pokhilenko) [inline api-assignment/petStoreAPITests1.md:4]: I see **post**, where is the **get**?

File: api-assignment/petStoreAPITests1.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
+For testing these scenarios we will need:
```
Comment (Pokhilenko) [inline api-assignment/petStoreAPITests1.md:1]: Before writing actual test scenarios and steps it's better to add some title like "Pet Store API - Order"

File: api-assignment/petStoreAPITests1.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
+For testing these scenarios we will need: 
+1.Postman or similar tool to perform HTTP requests 
+
+---post---
+
+Scenario/Case 1 - Create store order
```
Comment (Pokhilenko) [inline api-assignment/petStoreAPITests1.md:6]: Since you're using .md you can make it much more prettier by prefixing `####` for example

**Review by Pokhilenko — CHANGES_REQUESTED** [review]
Overall is good, but, please, make it little bit prettier and more readable by playing with .md styling.

**Comment by Pokhilenko** [comment]
resolve conflicts before merging and remove all the files which are not related to your current task

**Comment by neojan84** [comment]
I've deleted all the irrelevant files

**Pull request closed without merging**